### PR TITLE
[ECOM-5539] Add GTM script to store UTM querystring params in a cookie

### PR DESCRIPTION
**Pull request by bderusha**
@edx/ecommerce please review

### CODE REVIEW

File: gtm_scripts/utm_tracking/stage_utm_cookie.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            var aQueryStringParts = aQueryStringPairs[i].split("=");
+            if (sParameterName.toLowerCase() == aQueryStringParts[0].toLowerCase()) {
+                return aQueryStringParts[1];
+            }
+        }
+    };
+
+    var _setCookie = function (sCookieName, sCookieContents, iCookieLength) {
+        var dCookieExpires = new Date();
+        dCookieExpires.setTime(dCookieExpires.getTime() + (iCookieLength * 24 * 60 * 60 * 1000));
```
Comment (schenedx) [inline gtm_scripts/utm_tracking/stage_utm_cookie.js]: Can you put an variable for expression `24 * 60 * 60 * 1000`? Like `millisecondsInDay`?

**Comment by rlucioni** [comment]
@bderusha this is pretty new to me, so forgive me if the answer is otherwise obvious: is there a way to reduce the duplication we've got here? Most of the code in the two files touched here is the same.

**Comment by bderusha** [comment]
Unfortunately no.  These files are just to track the changes we are making directly in the GTM ui.  One for each environment

**Comment by bderusha** [comment]
The code is running on stage.edx.org if you want to check out the cookie


https://stage.edx.org/?utm_source=github&utm_medium=pr&utm_campaign=ECOM-5539

**Pull request merged**